mixins/StoreMixin.js

```diff
@@ -76,9 +76,16 @@ var StoreMixin = {
   store_addListeners: function () {
     Object.keys(this.store_listeners).forEach(function (storeID) {
       var listenerDetail = this.store_listeners[storeID];
+      var events = listenerDetail.events;
+      var eventKeys = Object.keys(events);
 
+      // Check that we actually have events to fire events on
+      if (!events || !eventKeys.length) {
```

Comment:
Also, it is handled here for when we add the store configs: https://github.com/dcos/dcos-ui/blob/master/src/js/plugin-bridge/PluginSDK.js#L202